src/Microsoft.ApplicationInsights.AspNet/project.json

```diff
@@ -14,7 +14,7 @@
         "delaySign": true
     },
     "dependencies": {
-        "Microsoft.ApplicationInsights": "2.0.0-*",
+        "Microsoft.ApplicationInsights": "2.0.0-beta4",
```

Comment:
Thank-you for all of your changes, unless there is a problem with 2.0.0-* please go ahead and change it and then we will complete the pull request.